client/components/gsuite/gsuite-price/index.jsx

```diff
@@ -54,7 +54,7 @@ const GSuitePrice = ( { currencyCode, product } ) => {
 	return (
 		<div className="gsuite-price">
 			<h4 className="gsuite-price__monthly-price">
-				{ translate( '{{strong}}%(price)s{{/strong}} per user/month', {
+				{ translate( '{{strong}}%(price)s{{/strong}} /user /month', {
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **24** times:
`translate( '{{strong}}%(price)s{{/strong}} per user / month' )` *ES Score: 7*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/upsell-for-google-workspace)